src/site/content/en/building-a-split-button-component/index.md

```diff
@@ -174,9 +174,10 @@ Add the primary button and the `.gui-popup-button` elements.
   <span class="gui-popup-button" aria-haspopup="true" aria-expanded="false" title="Open for more actions"></span>
 </div>
 ```
-
-Notice the aria attributes `aria-haspopup` and `aria-expanded`. These cues are
-critical for screen readers to be aware of the capability and state of split
+Notice that `.gui-popup-button` element is not of type/role **button** as it only
+interacts upon focus and no click events are needed.
```

Comment:
```suggestion
interacts up focus and no click events are needed.
```

Reply:
upon is fine here imo

Reply:
Simpler constructions are a best practice in writing.

Reply:
up focus doesn't make sense here, upon is correct.

Reply:
I meant it to be 'on'.